ci/create-tag-file.yml

```diff
@@ -0,0 +1,79 @@
+---
+#
+# (c) Copyright IBM Corp. 2021
+# (c) Copyright Instana Inc.
+#
+
+# see https://concourse-ci.org/tasks.html
+platform: linux
+image_resource:
+  type: registry-image
+  source:
+    repository: registry.access.redhat.com/ubi8/ubi
+outputs:
+  - name: message
+  - name: version
+params:
+  COMMIT_SHA: ((commit-sha))
+  CLASSIFIER: ((classifier))
+  VERSION: ((version))
+  FLAVOR: ((flavor))
+  RELEASE_TREE: ((release-tree))
+run:
+  path: /bin/bash
+  args:
+  - -exc
```

Comment:
shouldn't this be `-exec` ?

Reply:
(or what are the parameters for?)

Reply:
* -e is for exiting as soon as a command in the script exits with non-zero
* -x is for printing commands & params as they execute
* -c is for reading the script from stdin, which the following yaml list entry provides

thats at leats my understanding...